domaintools_phisheye/icon_domaintools_phisheye/connection/connection.py

```diff
@@ -0,0 +1,45 @@
+from domaintools import API
+from domaintools.exceptions import NotAuthorizedException
+from komand.exceptions import PluginException
+
+import komand
+from komand.exceptions import ConnectionTestException
+from .schema import ConnectionSchema, Input
+from icon_domaintools_phisheye.util.helper import Helper
+
+
+class Connection(komand.Connection):
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+        self.api = None
+        self.terms = None
+
+    def connect(self, params={}):
+        self.logger.info("Connect: Connecting..")
+        username = params.get(Input.USERNAME)
+        key = params.get(Input.API_KEY).get('secretKey')
```

Comment:
Utilize consistent quotation style across your code. I'd recommend sticking to double quotes